web/src/routes/App.component.tsx

```diff
@@ -64,7 +71,10 @@ export type AppStateProps = {
 
 export type AppDispatchProps = {
   dispatch: any
-  setNavigationPreference: (preference: 'mouse' | 'trackpad') => void
+  setNavigationPreference: (preference: typeof MOUSE | typeof TRACKPAD) => void
```

Comment:
```suggestion
  setNavigationPreference: (preference: NavigationPreference) => void
```

There should just be a thing called `NavigationPreference` that expresses the `|`

Reply:
yeah the only reason i didnt add it before was because the file type of the reducer is js, so i cant actually define a union type in there

Reply:
I see. can you change it to ts?